test/unit_tests/braket/parametric/test_free_parameter.py

```diff
@@ -61,3 +61,39 @@ def test_sub_successful(free_parameter):
 
 def test_sub_wrong_param(free_parameter):
     assert free_parameter.subs({"alpha": 1}) == FreeParameter("theta")
+
+
+@pytest.mark.parametrize(
+    "name",
+    (
+        "a",
+        "b",
+        "q",
+        "bit",
+        "qubit",
+        "_a",
+        "\u03B8",
+        "a\u03B8",
+        "a123",
+        "z123",
+        "\u03B8\u03B8",
+        "\u03B8a1",
+    ),
+)
+def test_valid_names(name):
+    FreeParameter(name)
+
+
+@pytest.mark.parametrize(
+    "name",
+    (
+        "",
+        "1",
+        "__a",
+        "!",
+        "\u33B8",
+    ),
+)
+@pytest.mark.xfail(raises=ValueError, strict=True)
```

Comment:
Use ```with pytest.raises(ValueError)``` instead

Reply:
The SDK has been misusing the xfail. Its [intended purpose](https://docs.pytest.org/en/7.1.x/how-to/skipping.html) is more so for bug fixes or items that are not implemented.